src/osyris/plot/wrappers.py

```diff
@@ -126,3 +126,50 @@ def scatter(ax, x, y, data, **kwargs):
         return coll
     else:
         return ax.scatter(x, y, **default_args)
+
+def line_integral_convolution(ax, x, y, z, **kwargs):
+    """
+    Wrapper that plots a line integral convolution of a vector field.
+    Uses alpha blending to merge the LIC with a user-requested color.
+    """
+    import lic as lic
+    from matplotlib.cm import ScalarMappable
+    from matplotlib.colors import LogNorm
+
+    #compute line integral convolution
+    try:
+        lic_res = lic.lic(z[...,1], z[...,0], length=kwargs["length"])
+        del kwargs["length"]
+    except KeyError:
+        lic_res = lic.lic(z[...,1], z[...,0], length=30)
+
```

Comment:
"length" is now also an additional plotting parameter for the lic. Its default value is 30 which gives good results for most plots, but the user might want a different value at times.

Reply:
I would use a different approach here. I would explicitly list `length` in the arguments (and `color` as well for that matter), and simply give it a default value of 30. Then you only need a single line here.
Look at the `labels` argument in the `contour` wrapper for an example.